parcels/kernel.py

```diff
@@ -431,6 +449,7 @@ def execute(self, pset, endtime, dt, recovery=None, output_file=None, execute_on
 
         # while np.any(error_particles):
         while n_error > 0:
+            print(n_error)
```

Comment:
print statement can go?

Reply:
I see there's still this print statement here. Can this be removed before we merge?